geometry_msgs/Point[] points
int32 path_cmd
---
bool success